### PR TITLE
Added AppConfig class to allow configuring App.Background dynamically.

### PR DESCRIPTION
**Pull request by Duckers**
This can be used as an optimization.

This PR contains:
- [x] Changelog
- [X] Documentation

### CODE REVIEW

File: Source/Fuse.Nodes/AppConfig.uno [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            <App>
+                ...
+                <AppConfig Background="{foo}" />
+    */
+    public class AppConfig: Behavior
+    {
+        public float4 Background 
+        {
+            get { return AppBase.Current.Background; }
+            set { AppBase.Current.Background = value; }
```
Comment (kusma) [inline Source/Fuse.Nodes/AppConfig.uno]: This is not really awesome... If there's two `AppConfig`-instances, they start overwriting each other's settings...

Comment (nekronos) [inline Source/Fuse.Nodes/AppConfig.uno]: Mhm 🤔 Could we make a special case for the Background property on App?

Comment (yupferris) [inline Source/Fuse.Nodes/AppConfig.uno]: That was my reaction as well, but I have no idea what a better alternative might be.

Comment (kusma) [inline Source/Fuse.Nodes/AppConfig.uno]: My thinking is that we should make change `App` so it is a normal Node. This would be a big change, though.

Comment (yupferris) [inline Source/Fuse.Nodes/AppConfig.uno]: Yeah that sounds like the right approach long-term, but appears to be a big change indeed.

Comment (yupferris) [inline Source/Fuse.Nodes/AppConfig.uno]: Alternatively, is it absolutely necessary that App maintains the clear color for the app, or is there another construct that could be better suited for this? (ignoring backwards-compatibility)

Comment (kusma) [inline Source/Fuse.Nodes/AppConfig.uno]: @yupferris: Well, this is kind of a hole we've dug ourselves into. The color really belongs to an implicit `GraphicsView` directly inside the App-tag. However, that is not exposed, so the user can't easily change that. That `App` has a `Background`-property is meant to be a way of exposing this to the API-user.

**Comment by mortoray** [comment]
Closing as abandoned.

**Pull request closed without merging**